fuzzing/scripts/build/libpng.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Copyright 2018-2019 by
+# Armin Hasitzka.
+#
+# This file is part of the FreeType project, and may only be used, modified,
+# and distributed under the terms of the FreeType project license,
+# LICENSE.TXT.  By continuing to use, modify, or distribute this file you
+# indicate that you have read the license and understand and accept it
+# fully.
+
+dir="${PWD}"
+cd $( dirname $( readlink -f "${0}" ) ) # go to `/fuzzing/scripts/build'
+
+path_to_zlib=$( readlink -f "../../../external/zlib" )
+
+path_to_src=$( readlink -f "../../../external/libpng" )
+path_to_build="${path_to_src}/build"
+path_to_install="${path_to_src}/usr"
+
+if [[ "${#}" -lt "1" || "${1}" != "--no-init" ]]; then
+
+    git submodule update --init "${path_to_src}"
+
+    cd "${path_to_src}"
+
+    git clean -dfqx
+    git reset --hard
+    git rev-parse HEAD
+
+    # Disable logging via library build configuration control.
+    cat scripts/pnglibconf.dfa | \
+      sed -e "s/option STDIO/option STDIO disabled/" \
+          -e "s/option WARNING /option WARNING disabled/" \
+          -e "s/option WRITE enables WRITE_INT_FUNCTIONS/option WRITE disabled/" \
+      > scripts/pnglibconf.dfa.temp
+    mv scripts/pnglibconf.dfa.temp scripts/pnglibconf.dfa
```

Comment:
oh noo haha I believe there is not other way to get around this? 🙈 ... out of curiosity -- is there a specific reason why we wouldn't want to use `sed -i`?

Reply:
> maybe a comment about where this comes from

this sounds great I think; if you don't mind adding that I'd appreciated it 😊

Reply:
Added comment already up at line 32 with a link to the build file this was sourced from, so that if anything ever stops working here we know where to look.